db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/validate_quota.rb

```diff
@@ -59,7 +59,7 @@ def display_value(item, value, _precision)
   when value == 1
     "1 Byte"
   when value < KILO_SIZE
-    "%d Bytes" % (value)
```

Comment:
@billfitzgerald0120 @tinaafitz 
All of these conversions can be easily converted using
value.to_s(:human_size)

Which will convert it to a human readable format